OneCommunityGlobal/HighestGoodNetworkApp
# Xiao Create Shadow Consistency for the Dashboard

# Description
(PRIORITY LOW) Jae: Create Shadow Consistency for the Dashboard
a. Please add the shadow shown below to the rest of the buttons on that page: 
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/181981a4-c6f6-4858-a9bb-65d53975da93)
b. Definitely these two areas, but any other areas on this page you think they would look good too: 
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/600bad49-31d8-41e4-af77-0c6fca8a9031)

## Main changes explained:
Added style={{ boxShadow: '2px 2px 4px 1px lightgray', }} to "Add Intangible Time Entry" button, "Badge Report" button, and "CountDownTimer" box. Also added some padding between the three icons next to "Tasks and Timelogs" to make it look better.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log in as any user and verify that the changes have been made

## Screenshots or videos of changes:
After:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/3b0251f9-e840-4772-ac01-f1fac6509425)
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/7a89d33a-84ca-419a-a8f9-aa331eb3dd36)
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/c95e24b4-63e8-4c0c-853c-44ed422cd306)

## Note:
This is an easy PR to test. The only changes are made to the UI. If you have any other suggestions for improvement, please let me know.
